src/appShell/App/footer.scss

```diff
@@ -18,13 +16,14 @@
 
         h3 {
             color: #666 !important;
+            font-size:15px;
         }
 
         ul {
             line-height: 20px;
             color: #666 !important;
             list-style: none;
-            padding-inline-start: 0;
```

Comment:
I looked up padding-inline-start.  It's an "experimental technology" !!!!  It has a little beaker next to it.  "it maps to a physical padding depending on the element's writing mode"  huh???